CITATION.cff

```diff
@@ -0,0 +1,166 @@
+    website: https://github.com/effigies
+    email: effigies@gmail.com
+  - family-names: de la Vega
+    given-names: Alejandro
+    website: https://github.com/adelavega
+
+authors:
+  # TEMPLATE
+  # - family-names: XXX
+  #   given-names: XXX
+  #   affiliation: XXX
+  #   website: XXX
+  #   email: XXX@XXX.XXX
+  #   orcid: "https://orcid.org/XXXX-XXXX-XXXX-XXXX"
+
+  #  Sorted alphabetically except for the 1rst and last authors
+
+  - family-names: Markiewicz
+    given-names: Christopher J.
+    website: https://github.com/effigies
+    email: markiewicz@stanford.edu
+    orcid: https://orcid.org/0000-0002-6533-164X
+    affiliation: Stanford University
+
+  - family-names: Bansal
+    given-names: Shashank
+    website: https://shashankbansal6.github.io/
+    email: shashankbansal56@gmail.com
+
+  - family-names: Blair
+    given-names: Ross W. 
+    website: https://github.com/rwblair
+
+  - given-names: Katherine
+    family-names: Bottenhorn
+
+  - family-names: Chen
+    given-names: Gang
+    website: https://github.com/afni-gangc
+    orcid: https://orcid.org/0000-0002-2960-089X
+    affiliation:
+      National Institutes of Health, Scientific and Statistical Computing Core,
+      NIMH, Bethesda, MD 20892, USA
+    email: gangchen@mail.nih.gov
+
+  - family-names: Dafflon
+    given-names: Jessica
+    website: https://jessicadafflon.github.io/
+
+  - family-names: Nielson
+    given-names: Dylan
+    affiliation:
+      Machine Learning Team at the National Institute of Mental Health
+    website: https://github.com/Shotgunosine
+
+  - family-names: Esteban
+    given-names: Oscar
+    website: http://www.axonlab.org
+    orcid: https://orcid.org/0000-0001-8435-6191
+    affiliation:
+      Department of Radiology, Lausanne University Hospital, Lausanne, CH-1011,
+      Switzerland
+    email: phd@oscaresteban.es
+
+  - family-names: Gau
+    given-names: Rémi
+    affiliation: Université catholique de Louvain
+    email: remi.gau@uclouvain.be
+    orcid: "https://orcid.org/0000-0002-1535-9767"
+
+  - family-names: Kent
+    given-names: James
+    affiliation:
+      Psychoinformatics Laboratory at the University of Texas at Austin
+    website: https://jdkent.github.io/
+    email: jamesdkent21@gmail.com
+
+  - given-names: Camille
+    family-names: Maumet
+    website: https://camillemaumet.com/
+    orcid: https://orcid.org/0000-0002-6290-553X
+    affiliation: Inria, Univ Rennes, CNRS, Inserm, Rennes, 35042, France
+
+  - family-names: Mumford
+    given-names: Jeanette
+    website: https://github.com/jmumford
+    email: jeanette.mumford@gmail.com
+
+  - family-names: Narayan
+    given-names: Manjari
+    email: manjari@alumni.rice.edu
+
+  - family-names: Nichols
+    given-names: Thomas E. 
+    website: http://nisox.org
+    orcid: https://orcid.org/0000-0002-4516-5103
+    affiliation:
+      "University of Oxford, Big Data Institute, Oxford, OX3 7LF, UK "
+    email: thomas.nichols@BDI.OX.AC.UK
```

Comment:
@nicholst This will be used to credit you as a co-author of the BIDS Stats Models specification. Please confirm that you would like to be listed as an author and update the metadata as you see fit.

Reply:
Approved!  Only one thing: @effigies: I appear to be the only one where the affiliation is quoted text, and, if it matters, there is a trailing space in the string.  Should we remove the quotes to match all the other entries?

Reply:
@remi-gau might have a reason, but I don't know it. I think reformatting to be a single line is probably fine.

Reply:
I don't have a reason, but there is a reason. The script I used to generate those citation.cff files takes as input the BIDS contributor form you filled in. And there was an extra space in your affiliation there. :-)
Will fix the script to avoid that in the future.